common/app/views/fragments/email/stylesheets/front.scala.html

```diff
@@ -45,14 +45,17 @@
         text-decoration: none;
     }
     .facia-card__text {
-        padding: 0 10px 26px 10px !important;
+        padding: 0 10px 0px 10px !important;
```

Comment:
That is sad! Would you be able to add a comment on these lines to explain that? It would be awful for the next person that works on emails to go in an delete them on because the "appear" not to be needed.